src/ws/Socket.ts

```diff
@@ -592,6 +592,9 @@ export class Socket extends EventEmitter {
      * from a chat server. Pass in Events to be opted out from as args
      */
     public optOutEvents(args: string[]): Promise<void> {
+        if (args.length === 0) {
+            return Promise.resolve();
```

Comment:
This feature was implemented keeping in mind Mixer first party clients and currently we'll only support opting out. Meaning, if you send the same method twice chat server will add to the list of events to opt out from if doesn't exist otherwise `noop`. If this is required in future we can add that feature.

Reply:
Also, when user reconnects they can choose not to send this method and will be opting in automatically for all events